### ISSUE TITLE
Mention inotify requirement in Docs

### ISSUE THREAD

**Issue by mitsuhiko** [lektor/lektor#93]
Some linux users with more esoteric file systems are reporting problems with FS notifications. It should be mentioned in the docs that Lektor in many ways requires working file system notifications.

This is as far as I know only an issue on linux as FS events work fine everywhere else.

**Comment by bameda** [comment]
It's happend with
- Arch linux + XFS filesystem + Vim editor.
- ~~Ubuntu + EXT4 filesystem + Atom editor~~  (to confirm)

The `webpack/webpack.config.js`

``` js
  var webpack = require('webpack');
  var path = require('path');
  var ExtractTextPlugin = require('extract-text-webpack-plugin');

  module.exports = {
      entry: {
          'app': './js/main.js',
          'styles': './scss/main.scss'
      },
      output: {
          path: path.dirname(__dirname) + '/assets/static/gen',
          filename: '[name].js'
      },
      devtool: '#cheap-module-source-map',
      resolve: {
          modulesDirectories: ['node_modules'],
          extensions: ['', '.js']
      },
      module: {
          loaders: [
              {
                  test: /\.js$/, exclude: /node_modules/,
                  loader: 'babel-loader'
              },
              {
                  test: /\.scss$/,
                  loader: ExtractTextPlugin.extract('style-loader', 'css-loader!sass-loader')
              },
              {
                  test: /\.css$/,
                  loader: ExtractTextPlugin.extract('style-loader', 'css-loader')
              },
              {
                  test: /\.(woff2?|ttf|eot|svg|png|jpe?g|gif)$/,
                  loader: 'file'
              }
          ]
      },
      plugins: [
          new ExtractTextPlugin('styles.css', { allChunks: true }),
          new webpack.optimize.UglifyJsPlugin()
      ]
  };
```

The `webpack/package.json`

``` js
  {
     "private": true,
     "devDependencies": {
       "babel-core": "^6.4.0",
       "babel-loader": "^6.2.1",
       "css-loader": "^0.23.1",
       "extract-text-webpack-plugin": "^0.9.1",
       "file-loader": "^0.8.5",
       "node-sass": "^3.4.2",
       "sass-loader": "^3.1.2",
       "style-loader": "^0.13.0",
       "url-loader": "^0.5.7",
       "webpack": "^1.12.11"
     }
  }
```

Some webpack related issues with some workarounds: https://github.com/webpack/webpack/issues/781 

To solve it I use `OldWatchingPlugin`: Edit `webpack/webpack.config.js` and add:

```
      ...
      plugins: [
          ...
          new webpack.OldWatchingPlugin()
      ]
  };
```

**Comment by soltysh** [comment]
This happened to me with Fedora 23, I don't see this OS as esoteric. What is the necessary limit?

**Comment by go-bears** [comment]
I think we can do this ticket. Would we make this notification for linux users on the installation page?
https://github.com/lektor/lektor-website/blob/master/content/docs/installation/contents.lr

**Comment by dairiki** [comment]
I believe this was fixed by #886 (and remained fixed through #1136).
Closing.
